src/reducers/index.js

```diff
@@ -987,20 +983,6 @@ const onSetFeatureToggles = (state, { payload }) => ({
   featureToggles: payload,
 });
 
-/**
- * @param {Object} action.payload The unleash client to store
- */
-const onSetUnleashClient = (state, { payload }) => ({
-  ...state,
-  unleashClient: payload,
-});
-
-const onWalletResetSuccess = (state) => ({
-  ...state,
-  // Keep the unleashClient as it should continue running
-  unleashClient: state.unleashClient,
-});
```

Comment:
By deleting the `unleashClient`,  this function would only return the `state` unmodified. So, it was completely removed along with its related actions.